### PR TITLE
SNOW-930831: Add randomization to the table name in put/get test

### PR DESCRIPTION
**Pull request by sfc-gh-pmotacki**
### Description
Please explain the changes you made here.

### Checklist
- [ ] Format code according to the existing code style (run `npm run lint:check -- CHANGED_FILES` and fix problems in changed code)
- [ ] Create tests which fail without the change (if possible)
- [ ] Make all tests (unit and integration) pass (`npm run test:unit` and `npm run test:integration`)
- [ ] Extend the README / documentation and ensure is properly displayed (if necessary)
- [ ] Provide JIRA issue id (if possible) or GitHub issue id in commit message

### CODE REVIEW

**Review by sfc-gh-dheyman-1 — APPROVED** [review]

File: test/integration/testConnectionPoolExecute.js [hunk 1]
```diff
@@ -4,14 +4,16 @@
...
+const {randomizeName} = require('./testUtil');
 
 describe('Execute test with Pool', function () {
   let connectionPool = null;
-  const createNodeTSQL = 'create or replace table NodeT(colA number, colB varchar)';
-  const selectAllSQL = 'select * from NodeT';
-  const insertNodeTSQL = 'insert into NodeT values(1, \'a\')';
-  const updateNodeTSQL = 'update NodeT set COLA = 2, COLB = \'b\' where COLA = 1';
-  const dropNodeTSQL = 'drop table if exists NodeT';
+  const nodeT= randomizeName('nodeT');
```
Comment (sfc-gh-pfus) [inline test/integration/testConnectionPoolExecute.js:11]: Do you have schema creation per job? If yes, what is the purpose of these changes?

Comment (sfc-gh-pmotacki) [reply test/integration/testConnectionPoolExecute.js:11]: Yes schema is randomised per job but still randomising of table name can protect from parallel running by mocha. It isn't used in ci scripts by every one can use it running scripts.

**Review by sfc-gh-pfus — APPROVED** [review]
